install/kubernetes/cilium/values.yaml

```diff
@@ -1531,8 +1531,12 @@ nodePort:
 policyEnforcementMode: "default"
 
 pprof:
-  # -- Enable Go pprof debugging
+  # -- Enable pprof for cilium components
```

Comment:
This comment implies the configuration is used for all components, but I don't think that's true? Because cilium-agent and cilium-operator run in the same network namespace (i.e. host ns), they are using a different port (6600 vs 6061). Hubble Relay also uses a different port (6062), but I think that's not strictly necessary (since it's running in its own network ns)

We should clearly document that this port is only used for the agent. Maybe it's even worth adding additional Helm flags for the operator and relay too?

Reply:
Alright, I've broken out the configuration of pprof for each component. I've opted to keep the shared cilium-config configmap to ease back porting this PR.